Mathlib/Order/PartialSups.lean

```diff
@@ -3,188 +3,280 @@ Copyright (c) 2021 Kim Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kim Morrison
 -/
-import Mathlib.Data.Finset.Lattice.Fold
+import Mathlib.Algebra.Order.SuccPred
```

Comment:
Could you move to a new file `Algebra.Order.PartialSups` everything that depends on this new import?

Reply:
I have done this (awaiting CI and the inevitable `shake` failures). However, for now I need to import from this new file from `Algebra.Order` in `Order.Disjointed`; this is a **temporary measure** since I already have a PR in the works which will completely overhaul `Order.Disjointed` anyway, and I will ensure that nothing from `Algebra.Order` gets imported in `Order` once that PR is in.